docs/developer-guide/nodejs-api.md

```diff
@@ -220,21 +220,20 @@ The return value is an object containing the results of the linting operation. H
 {
     results: [
         {
-            filePath: "./myfile.js",
-            output: "foo;",
-            messages: [
-                {
-                    fatal: false,
-                    severity: 2,
-                    ruleId: "semi",
-                    severity: 2,
-                    line: 1,
-                    column: 23,
-                    message: "Expected a semicolon."
-                }
-            ],
+            filePath: "/Users/eslint/project/myfile.js",
+            messages: [{
+                ruleId: "semi",
```

Comment:
This is just `JSON.stringify` with removed quotes from keys. Should the order match the old order?